spec/features/editing_images/upload_inline_image_spec.rb

```diff
@@ -5,7 +5,7 @@
     given_there_is_a_document
     when_i_visit_the_images_page
     and_i_upload_a_new_image
-    then_it_has_a_normalised_filename
+    then_i_see_it_has_a_snippet
```

Comment:
Is there a missing step that checks that the user has the correct permissions to see the snippet?